tools/qcxms/qcxms_prod_run.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="qcxms_prod_run" name="QCxMS prod run" version="@TOOL_VERSION@+galaxy0" python_template_version="3.5" profile="21.05">
+    <description>production runs</description>
+    
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+
+    <expand macro="creator"/>
+
+    <edam_topics>
+        <edam_topic>topic_3332</edam_topic>
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_0297</edam_operation>
+    </edam_operations>
+
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">qcxms</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        python3 ${create_folder} &&
+        find TMPQCXMS/*/ -type d | xargs -I {} -P 4 sh -c 'cd {} && qcxms --prod >> $log' &&
+        getres &&
+        plotms
+    ]]></command>
+
+    <environment_variables>
+        <environment_variable name="OMP_NUM_THREADS">1</environment_variable>
+    </environment_variables>
+
+    <configfiles>
+        <configfile name="create_folder">
+import os
+import shutil
+
+#set folder1_files = str("', '").join([str($f) for $f in $in_files])
+#set folder2_files = str("', '").join([str($f) for $f in $start_files])
+#set folder3_files = str("', '").join([str($f) for $f in $xyz_files])
+
+#set names = str("', '").join([str($f.name) for $f in $xyz_files])
+names = '$names'
+folder_names = [x.split("_")[0] for x in names]
+
+folder1_files = '$folder1_files'
+folder2_files = '$folder2_files'
+folder3_files = '$folder3_files'
```

Comment:
Variable names could be improved, like "input_config_files" etc.

Reply:
fixed in [77886de](https://github.com/RECETOX/galaxytools/pull/421/commits/77886de66a13a179bbbb88bbd5ade5689683dadb)